app/Http/Controllers/Admin/PermissionGroupController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Admin\PermissionGroupRequest;
+use App\Repositories\Admin\PermissionGroup\PermissionGroupRepositoryInterface as PermissionGroupRepository;
+
+class PermissionGroupController extends Controller
+{
+    protected $permissionGroupRepositories;
+
+    public function __construct(permissionGroupRepository $permissionGroupRepository)
+    {
+        $this->permissionGroupRepository = $permissionGroupRepository;
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view('admin.permission-group.index', [
+        'permissionGroups' => $this->permissionGroupRepository->paginate(),
+        ]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('admin.permission-group.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(PermissionGroupRequest $request)
+    {
+        $this->permissionGroupRepository->save($request->validated());
+
+        return redirect()->route('admin.permission-group.index');
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $permissionGroup = $this->permissionGroupRepository->findById($id);
```

Comment:
if (!$permissionGroup  = $this->permissionGroupRepository->save($request->validated())) {
    abort(404);
}